integration-tests/known_networks.go

```diff
@@ -1,103 +1,184 @@
 		Simulated:                 false,
-		PrivateKeys:               strings.Split(os.Getenv("EVM_PRIVATE_KEYS"), ","),
 		ChainlinkTransactionLimit: 5000,
 		Timeout:                   time.Minute,
 		MinimumConfirmations:      1,
 		GasEstimationBuffer:       0,
 	}
 
-	// MetisStardust https://www.metis.io/
+	// metisStardust https://www.metis.io/
 	MetisStardust *blockchain.EVMNetwork = &blockchain.EVMNetwork{
 		Name:                      "Metis Stardust",
 		ClientImplementation:      blockchain.MetisClientImplementation,
 		ChainID:                   588,
-		URLs:                      []string{"wss://stardust-ws.metis.io/"},
 		Simulated:                 false,
-		PrivateKeys:               strings.Split(os.Getenv("EVM_PRIVATE_KEYS"), ","),
 		ChainlinkTransactionLimit: 5000,
 		Timeout:                   time.Minute,
 		MinimumConfirmations:      1,
 		GasEstimationBuffer:       1000,
 	}
 
-	// ArbitrumGoerli https://developer.offchainlabs.com/docs/public_chains
+	// arbitrumGoerli https://developer.offchainlabs.com/docs/public_chains
 	ArbitrumGoerli *blockchain.EVMNetwork = &blockchain.EVMNetwork{
 		Name:                      "Arbitrum Goerli",
 		ClientImplementation:      blockchain.ArbitrumClientImplementation,
 		ChainID:                   421613,
-		URLs:                      strings.Split(os.Getenv("EVM_URLS"), ","),
 		Simulated:                 false,
-		PrivateKeys:               strings.Split(os.Getenv("EVM_PRIVATE_KEYS"), ","),
 		ChainlinkTransactionLimit: 5000,
 		Timeout:                   time.Minute,
 		MinimumConfirmations:      0,
 		GasEstimationBuffer:       0,
 	}
 
-	// OptimismGoerli https://dev.optimism.io/kovan-to-goerli/
+	// optimismGoerli https://dev.optimism.io/kovan-to-goerli/
 	OptimismGoerli *blockchain.EVMNetwork = &blockchain.EVMNetwork{
 		Name:                      "Optimism Goerli",
 		ClientImplementation:      blockchain.OptimismClientImplementation,
 		ChainID:                   420,
-		URLs:                      strings.Split(os.Getenv("EVM_URLS"), ","),
 		Simulated:                 false,
-		PrivateKeys:               strings.Split(os.Getenv("EVM_PRIVATE_KEYS"), ","),
 		ChainlinkTransactionLimit: 5000,
 		Timeout:                   time.Minute,
 		MinimumConfirmations:      0,
 		GasEstimationBuffer:       0,
 	}
+
+	mappedNetworks = map[string]*blockchain.EVMNetwork{
+		"SIMULATED":       SimulatedEVM,
+		"GENERAL":         generalEVM,
+		"GOERLI":          GoerliTestnet,
+		"SEPOLIA":         SepoliaTestnet,
+		"KLAYTN_BAOBAB":   KlaytnBaobab,
+		"METIS_STARDUST":  MetisStardust,
+		"ARBITRUM_GOERLI": ArbitrumGoerli,
+		"OPTIMISM_GOERLI": OptimismGoerli,
+	}
 )
+
+// determineSelectedNetworks uses `SELECTED_NETWORKS` to determine which network(s) to run the tests on
+func determineSelectedNetworks() []*blockchain.EVMNetwork {
+	logging.Init()
+	selectedNetworks := make([]*blockchain.EVMNetwork, 0)
+	setNetworkNames := strings.Split(strings.ToUpper(os.Getenv("SELECTED_NETWORKS")), ",")
```

Comment:
I thought envconfig could automatically create these slice's for us? Should be able to do the same for urls and keys which might simplify some of this logic.

Reply:
Ya, can move this to the chainlink-env logic.